### DISCUSSION TITLE
Chia windows gui do unneded copy (two copy of the pilot on some drive whenever doing) please rather the copy &delete temp pilot use move command much better.

### DISCUSSION THREAD

**Discussion by spamkutu** [Chia-Network/chia-blockchain#8388] [Ideas]
After complished Time for phase 4 like this below
Approximate working space used (without final file): 269.506 GiB
Final File size: 101.443 GiB

I suppose chia do unneded copy progress beside to some hdd and dosent mater diff hdd too, I see Its take 30min two 2hours waste on windows chia gui.  I offer you to please after phase 4 beside copy and delete pilot move pilot command will be much faster and no wast so many times.

![1](https://user-images.githubusercontent.com/13944520/132470936-ff269b0f-b649-4adf-bde0-882116a8ef7e.png)
![2](https://user-images.githubusercontent.com/13944520/132470944-23e835c5-5700-416a-ac08-1bc9ab77572b.png)

the time show me 11.09 am but still copy progress continoue whicever started about 10.25 am realy weird.

**Comment by BrandtH22** [comment]
@spamkutu , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

Note - when plotting be sure to use the temp and destination drive options for the plots to be automatically copied to the destination drive.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
